static/src/styles/NotFound.styl

```diff
@@ -0,0 +1,16 @@
+.NotFound_container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.NotFound_title {
```

Comment:
Nit: elements have _two_ underscores between block and element name (so `.NotFound__title`)

Reply:
@Pinpickle out of interest, why is it two vs one? It's not easy to tell the difference anyway. So I don't see how it helps?

Reply:
The underlying decisions are mostly arbitrary, but we chose it because it's a widely-adopted convention: https://github.com/airbnb/css#oocss-and-bem

It's much easier to say "we follow the Airbnb style guide" and let them justify everything, which they do quite well.